spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -617,7 +619,7 @@ public class BusinessResource {
     @PUT
     @Path("/after")
     @AfterLRA // method is called when the LRA associated with the method activityWithLRA finishes
-    public Response notifyLRAFinished(@HeaderParam(LRA_HTTP_ENDED_CONTEXT_HEADER) URI lraId,
+    public Response notifyLRAFinished(@HeaderParam(LRA_HTTP_ENDED_CONTEXT_HEADER) URI lraId, // LRA_HTTP_ENDED_CONTEXT_HEADER is used because this method is also allowed to start a new LRA
```

Comment:
I don't understand the comment about the method is allowed to start a new LRA. That sounds incorrect to me.

Reply:
I will rephrase, we use a different header because LRA_HTTP_CONTEXT_HEADER can contain newly started LRA at the afterLRA method.

Reply:
Thanks.